ams/CleanupCommand.cs

```diff
@@ -111,6 +106,7 @@ private async Task<bool> CleanUpAccountAsync(MediaServicesAccountResource accoun
                 {
                     foreach (var streamingEndpoint in endpoints)
                     {
+                        await streamingEndpoint.StopAsync(WaitUntil.Completed);
```

Comment:
Need to follow the same pattern as how to delete LiveEvent here,  Stop the StreamingEndpoint only when it is in running state.